kubernetes-model/src/main/resources/schema/validation-schema.json

```diff
@@ -13581,6 +13581,10 @@
           "$ref": "#/definitions/kubernetes_Container",
           "javaType": "io.fabric8.kubernetes.api.model.Container"
         },
+        "idleTimeout": {
```

Comment:
@ericbottard I'm not sure why this seemed to be duplicated on generation. I assume it's safe to remove one?

Reply:
I've never touched this file by hand. It is new following the rebase on latest fabric8 master (separate file for validation). I'm changing things there (pointer) anyway, let's see what happens. I'd refrain from manual tempering though